### PR TITLE
Deploying documentation to Github pages

### PR DESCRIPTION
**Pull request by genisplaja**
In this PR (sorry for the massive amount of commits) we basically include the components to deploy the documentation to github pages.

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+sphinx
+matplotlib>=3.0.0
+numpy==1.19.5
+librosa==0.8.0
+SoundFile==0.10.3.post1
+joblib==0.17.0
+pathlib==1.0.1
+pytsmod==0.3.3
```
Comment (thomasgnuttall) [inline docs/requirements.txt:8]: son los requirements para la vis o para todo?

**Review by thomasgnuttall — APPROVED** [review]

File: docs/requirements.txt (hunk 1 above)
Comment (genisplaja) [reply docs/requirements.txt:8]: creo que realmente esto no sirve para nada actually

Comment (genisplaja) [reply docs/requirements.txt:8]: let me make sure of that y me lo cargo

**Comment by genisplaja** [comment]
Right @thomasgnuttall that looks ready to go :)

**Review by thomasgnuttall — APPROVED** [review]

**Pull request merged**